synce_port_ctrl.c

```diff
@@ -822,6 +822,9 @@ int synce_port_ctrl_destroy(struct synce_port_ctrl *pc)
 	}
 	pr_debug("%s on %s", __func__, pc->name);
 
+	if (!pc->transport)
+		return 0;
```

Comment:
Would you be so kind to return `NULL` instead of `0`? I know it's equivalent, but rest of the code tries to follow NULL-convention. Thanks!

Reply:
This function is declared as returning integers and there are already two return statements returning an integer.

Reply:
Yes, you are right - my oversight - thanks.